app/services/iiif_print/derivative_rodeo_service_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# OVERRIDE IIIF Print v1.0.0 to only run thumbnail derivatives on thumbnail files
+
+module IiifPrint
+  module DerivativeRodeoServiceDecorator
+    def create_derivatives(filename)
```

Comment:
Sidenote:

- This has me thinking about a predicate function that we can add to IiifPrint configuration and reworking the method signature to allow for configuration of this functionality.